app/assets/stylesheets/homepage.scss

```diff
@@ -0,0 +1,40 @@
+$color: #EC4339;
+
+.title-box {
+   background-color: $color;
+   font-family: 'Alegreya Sans SC', sans-serif;
+   color: #ffffff;
```

Comment:
Color literals like `#ffffff` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Line should be indented 2 spaces, but was indented 3 spaces